common/views/components/SubNavigation/SubNavigation.styles.tsx

```diff
@@ -63,7 +63,7 @@ export const NavItemInner = styled.a.attrs<NavItemInnerProps>(props => {
 })<NavItemInnerProps>`
   display: block;
   position: relative;
-  padding-bottom: 1.5rem;
+  padding: 0 10px 24px;
```

Comment:
Maybe worth a comment about why we're adding this padding?

Reply:
In case someone tries to fix the alignment?

Reply:
More like "it's deliberately offset from the left-hand side to make the buttons bigger" – i.e. that there are more than just visual reasons for this choice, which isn't the case in a lot of places